src/main/webapp/config/scripts/default/CreateTables.sql

```diff
@@ -1,35 +1,30 @@
 CREATE TABLE users (
```

Comment:
LGTM.

Maybe as future step: we have the database structure in a file, the queries and statements in code. Maybe it makes sense to have everything at one place, otherwise noone finds the stuff (complaints in ADAP) - or we just need to document it in the readme..